recipe/meta.yaml

```diff
@@ -28,16 +28,19 @@ build:
 requirements:
   build:
     - {{ posix }}zip               # [win]
+    - make
```

Comment:
```suggestion
    - make
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
```

And then you'll need to rerender

Reply:
mostly just taken from the dplyr conda-forge recipe

Reply:
@mariusvniekerk thank you for the hint and suggestion!